components/innerHeader.tsx

```diff
@@ -2,20 +2,23 @@ import { Box, Button, H1, HR, Text } from '@bigcommerce/big-design';
 import { ArrowBackIcon } from '@bigcommerce/big-design-icons';
 import { useRouter } from 'next/router';
 import { useProductList } from '../lib/hooks';
-import { TabIds, TabRoutes } from './header';
+import { RouteTitles, TabRoutes } from './header';
 
 const InnerHeader = () => {
     const router = useRouter();
     const { pid } = router.query;
     const { list = [] } = useProductList();
     const { name } = list.find(item => item.id === Number(pid)) ?? {};
 
-    const handleBackClick = () => router.push(TabRoutes[TabIds.PRODUCTS]);
+    const routeHead = router.pathname.split('/')[1];
+    const title = RouteTitles[routeHead];
+
+    const handleBackClick = () => router.push(TabRoutes[routeHead]);
```

Comment:
since `TabRoutes` is just the route with a forward slash (and given orders isn't actually a tabbed route), we could probably just eliminate the use of `TabRoute` here.
```jsx
    const basePath = router.pathname.split('/')[1];
    const handleBackClick = () => router.push(`/${basePath}`);
```

Reply:
This actually brings up a question about where we want this button to direct to. When on `/orders/[orderId]` the text says `Orders` But there isn't currently an orders page to navigate to. If we do plan to include an orders page at some point, it would make sense to keep the label but maybe have it navigate to the home page for now. If we don't intend on adding an orders page, it might make more sense to rename it or disable it like the "Create shipment" button.